src/features/txnDelegatation/useHeliumTxns.ts

```diff
@@ -0,0 +1,136 @@
+import { AddGateway, Location, Transfer } from '@helium/react-native-sdk'
+import { SignHotspotResponse } from '@helium/wallet-link'
```

Comment:
Think theres a typo in the folder name. `txnDelegatation` => `txnDelegation`

Reply:
Doh. Thanks.